Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1098,40 +1111,33 @@ def fetch_incidents(client, first_fetch_time, integration_instance, exclude_arti
         demisto.debug(
             f"starred_incidents_fetch_window after parsing date range {starred_incidents_fetch_window}")
 
-    incidents = []
     if incidents_from_previous_run:
+        demisto.debug('Using incidents from last run')
```

Comment:
I thought we want to remove this

Reply:
We removed this from common use, but it's still used when there is a rate-limit whilst enriching incidents with >50 alerts